### ISSUE TITLE
Set Initial State

### ISSUE THREAD

**Issue by Nightbl927** [sonaye/react-native-micro-animated-button#13]
Is there a way you could add the ability to set the initial state to success as an option?  I have a project where if an item was already completed, I would like to set the state to a checkmark initially.

**Comment by sonaye** [comment]
Right now you can `<Btn ref={ref => this.state.success && ref.success()} />`, but I'll look into adding an initial state without animation, is that what you are looking for? Also, see #2.

**Comment by Nightbl927** [comment]
Thanks for the quick answer.  Yes, an initial state without the animation so as soon as the component loads, it appears in the success state already.

**Comment by sonaye** [comment]
Released in `0.0.20`.
```javascript
<Btn initialState="success" />
<Btn initialState="error" />
<Btn initialState="loading" />
```
